index.html

```diff
@@ -26,5 +27,121 @@ <h2 class="intro-section-item" id="intro-text2">I’am a Software Developer</h2>
       <li><a href="" class="icon-element" type="button"><img src="src/m.png" alt="m"></a></li>
     </ul>
   </section>
+
+  <section class="recent-works">
+    <h1 class="works">My Recent Works</h1>
+    <img class="works-line" src="src/line.png" alt="line">
+
+    <div class="project-container project1">
+      <img class="project-item project-image" src="src/project-image.png" alt="project-image">
+      <h1 class="project-item project-name">Multi-Post Stories</h1>
+      <p class="project-item project-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="project-item lang-container">
+        <li class="lang-item lang-css">css</li>
+        <li class="lang-item lang-html">html</li>
+        <li class="lang-item lang-bootstrap">bootstrap</li>
+        <li class="lang-item lang-ruby">Ruby</li>
+      </ul>
+      <button type="button" class="project-item project-button buttons">See Project</button> 
+    </div>
+
+    <section class="card-grid-container">
+      <div class="card-project-container card-grid-item1">
+        <h1 class="card-project-item card-project-title">Professional Art Printing Data</h1>
```

Comment:
I suggest you change this to any other heading `<hx> from <h2> to <h6>`. Your document should only have one `<h1>` element. Your browsers uses this for seo optimization and also for your site to be used by screen readers. Read [here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.) for more on this